### PR TITLE
Add Club/Fanclub/Footballclub structures

### PR DESCRIPTION
**Pull request by fikusmaksymilian**
Added classes for Club, Fanclub and Footballclub

### CODE REVIEW

**Review by kejmil4 — APPROVED** [review]
code fine

File: FMSim/FMBase/Clubs/Fanclub.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace FMBase.Clubs;
+
+public class Fanclub(string name, string description, string colors, int[] members, string loadout, string balance) : Club(name, description, colors, members)
+{
+    public string Loadout = loadout;
```
Comment (MishutkAviation) [inline FMSim/FMBase/Clubs/Fanclub.cs:5]: what is loadout

Comment (fikusmaksymilian) [reply FMSim/FMBase/Clubs/Fanclub.cs:5]: Loadout is sprzęt my brother (Ak-47, MAC-10)

File: FMSim/FMBase/Clubs/FootballClub.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace FMBase.Clubs;
+
+public class FootballClub(string name, string description, string colors, int[] members, string formation, string jersey) : Club(name, description, colors, members)
+{
+    public string Formation = formation;
```
Comment (MishutkAviation) [inline FMSim/FMBase/Clubs/FootballClub.cs]: Formation already implemented in Trainer class.

**Review by MishutkAviation — CHANGES_REQUESTED** [review]
Changes pending in comment.

File: FMSim/FMBase/Clubs/FootballClub.cs (hunk 2 above)
Comment (fikusmaksymilian) [reply FMSim/FMBase/Clubs/FootballClub.cs]: Formation in Trainer is problem not good, make it in Club and fine, not my problem go ask Jan G.

Comment (MishutkAviation) [reply FMSim/FMBase/Clubs/FootballClub.cs]: Implementation in Trainer was already confirmed, please remove this field from FootballClass.

**Review by MishutkAviation — APPROVED** [review]
Pending changes fixed, PR approved

**Pull request merged**